sdcm/cluster.py

```diff
@@ -3042,7 +3039,7 @@ def terminate_node(self, node):
         node.destroy()
 
     def get_db_auth(self):
-        if self.params.get('use_ldap_authorization') and self.ldap_configured:
+        if self.params.get('use_ldap_authorization') and self.params.get('are_ldap_users_on_scylla'):
```

Comment:
Is this a new param?
Is it in sct_config?
If new, change this name.

Reply:
it is a new param, but it is used only as a flag, to know when we will be able to use the LDAP users (only after they were created in Scylla using CQL)..
it is not in sct_config, as it is not configurable by test config file

Reply:
It's not good to use a new parameter as a flag, which can't be assigned in configure file.